bugbug/repository.py

```diff
@@ -17,7 +17,7 @@
 import hglib
 import requests
 from dateutil.relativedelta import relativedelta
-from parsepatch.patch import Patch
+import rs_parsepatch as rs_pp
```

Comment:
Nit: I'd prefer a more explanatory name, e.g. leave it as `rs_parsepatch`, or `parsepatch`